client/js/components/procedure/SegmentsList/SegmentsList.vue

```diff
@@ -983,6 +985,13 @@ export default {
       lscache.set(this.lsKey.allSegments, allSegments)
     },
 
+    storeFilterInCache () {
```

Comment:
we have several options for storing data in the FE: localStorage, sessionStorage, or cache. You decided to store the values in the cache. hm, but i am not sure this is the best approach and we normally use localStorage for filters. What does gpt say:

**When to use what?**

**localStorage**

- Persists after reload, closing the tab, and restarting the browser.
- Perfect for: filters, user preferences, UI settings.

**sessionStorage**

- Only applies to the current browser tab session.
- If the tab is closed → the data is gone.
- Only useful if you explicitly do not want to store data beyond a single tab session.

**Cache**

- Browser cache is meant for assets/HTTP resources.
- Not a good place for storing your UI state.